fastlane/Fastfile

```diff
@@ -0,0 +1,33 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+
+default_platform(:android)
+
+platform :android do
+  desc "Build and deploy on Google Play"
+  lane :deploy do |options|
+    gradleTaskName = "assemble#{options[:flavor]}Release"
+    version = ENV['TRAVIS_TAG'].empty? ? 'SNAPSHOT' : ENV['TRAVIS_TAG']
+    package_name = options[:flavor] == 'unbranded' ? "org.medicmobile.webapp.mobile" : "org.medicmobile.webapp.mobile.#{options[:flavor]}"
+
+    gradle(task: gradleTaskName)
+
+    supply(
+      package_name: package_name,
+      track: "alpha",
+      json_key: "playstore-secret.json",
+      apk_paths: [
+        "build/outputs/apk/#{options[:flavor]}/release/medic-android-#{version}-#{options[:flavor]}-arm64-v8a-release.apk",
+        "build/outputs/apk/#{options[:flavor]}/release/medic-android-#{version}-#{options[:flavor]}-armeabi-v7a-release.apk",
+      ],
+      skip_upload_aab: true,
+      skip_upload_metadata: true,
+      skip_upload_images: true,
+      skip_upload_screenshots: true,
+      validate_only: false,
+
+      timeout: 3600,
+    )
+  end
+end
```

Comment:
I see fastlane also does signing - is it worth using it for that too?

Reply:
Saw that ... same for deploying to GitHub releases...  I recon a solid argument could be made for consolidating as much build logic as possible in a single system. Would certainly improve readability. 

I thought it was reasonable to keep it in gradle just because a) it's done and vetted and tested, b) this is my first experience with fastlane and i wasn't really ready to bet on it as the single point of consolidation, and c) keeping it in gradle allows us to keep a single, self-installing tool as the only requirement for the dev workflow.